packages/x-license/src/verifyLicense/verifyLicense.ts

```diff
@@ -99,10 +106,12 @@ export function verifyLicense({
   releaseInfo,
   licenseKey,
   acceptedScopes,
+  productScope = null,
```

Comment:
When do we call `verifyLicense` without a product scope?

Reply:
This is just for the cases where we cannot extract the product scope from the package ... this should ideally never happen, but as mentioned before, the `regex.match` has it in the return type, so this is basically to satisfy TS

Reply:
oh, i misread that
we should never ... sry about the comment above! :P